src/preact/base-element.js

```diff
@@ -406,7 +417,7 @@ export class PreactBaseElement extends AMP.BaseElement {
    */
   checkMutations_(records) {
     const Ctor = this.constructor;
-    this.mutationObserverCallback();
+    this.mutationObserverCallback(records);
```

Comment:
This should be fine, as the only other Component that uses mutationObserverCallback is base-carousel and it does so w/o any arguments.